ipython/ResamplingAndMonteCarloMethods/resampling_tutorial_2.ipynb

```diff
@@ -350,7 +358,7 @@
    "id": "459d7ca3-f719-4b8e-a855-8e47846c3ec0",
    "metadata": {},
    "source": [
-    "Note that `1` is added to both the numerator and denominator when performing the randomized test [[3]](https://www.degruyter.com/document/doi/10.2202/1544-6115.1585/html). This can be thought of as including the observed value of the test statistic in the null distribution, and it ensures that the $p$-value of a randomized test is never zero."
+    "Note that `1` is added to both the numerator and denominator when performing the randomized test [[6]](https://www.degruyter.com/document/doi/10.2202/1544-6115.1585/html). This can be thought of as including the observed value of the test statistic in the null distribution, and it ensures that the $p$-value of a randomized test is never zero."
```

Comment:
I believe that these references were originally numbered sequentially throughout the whole tutorial. There is no reference [1] in here, and [2] is a footnote. LMK know what you think we should do.

Reply:
I see, I ended up adding references and then trying to make them consistent again and then wasn't clear on what to do with the order. Maybe footnotes shouldn't get numbers, but like a symbol instead from `*, †, ‡, §, ‖, ¶`? And then references be numbered sequentially throughout?

Reply:
Yes, please!